src/core/unit.c

```diff
@@ -1340,13 +1340,29 @@ const char *unit_description(Unit *u) {
         return strna(u->id);
 }
 
-const char *unit_status_string(Unit *u) {
+const char* unit_status_string(Unit *u, char **combined) {
         assert(u);
+        assert(u->id);
 
-        if (u->manager->status_unit_format == STATUS_UNIT_FORMAT_NAME && u->id)
+        /* Return u->id, u->description, or "{u->id} ({u->description})".
```

Comment:
Maybe this has already been discussed, and maybe it is only me that thinks it this way, but I think a `"{u->description} ({u->id})"` format would be better in most cases.

Reply:
i agree with @dnicolodi. The boot-time output is one of the most user-facing thing we have, i am pretty sure we should output the nice strings there primarily, and the usable strings secondarily, i.e. in ()

Reply:
while I personally prefer the unit name first, having the unit names second and in `()` at least has the benefit of avoiding nested braces like in `(Userspace Out-Of-Memory (OOM) Killer)`. 
`{u->id}: {u->description}` looks a bit more natural too me. 

That said, I'm well aware that this is bike shedding territory and if in doubt would go with the existing implementation.

Reply:
> `{u->id}: {u->description}` looks a bit more natural too me.

or `{u->id} - {u->description}`. The benefit of this would be, that it mirrors the `systemctl status` output

Reply:
I'll change it to either `{u->id}: {u->description}` or `{u->id} - {u->description}` (I want to test it to see how it looks in practice.)

I think it's important to have the id first, because descriptions are often very long and will get truncated.

Reply:
I think the version with " - " looks marginally better. I also like that it mirrors `status`.